src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1537,11 +1546,14 @@ private void removeParticipantIdentifierFromList(
             }
             List<String> possibleRecipientsForRemainingGiver =
                                             bundle.getPossibleRecipients(question, possibleGiverWithNoResponses);
-            
-            responseRows.addAll(buildMissingResponseRowsBetweenGiverAndPossibleRecipients(
-                                            question, possibleRecipientsForRemainingGiver, possibleGiverWithNoResponses,
-                                            bundle.getFullNameFromRoster(possibleGiverWithNoResponses),
-                                            bundle.getTeamNameFromRoster(possibleGiverWithNoResponses)));
+            if (isMissingResponsesShown) {
+                responseRows.addAll(buildMissingResponseRowsBetweenGiverAndPossibleRecipients(
+                                    question,
+                                    possibleRecipientsForRemainingGiver,
+                                    possibleGiverWithNoResponses,
+                                    bundle.getFullNameFromRoster(possibleGiverWithNoResponses),
+                                    bundle.getTeamNameFromRoster(possibleGiverWithNoResponses)));
```

Comment:
Here it looks like `question` onwards are params of `responseRows.addAll`. One way to make it better:
```java
responseRows.addAll(
        buildMissingResponse...(
                question, possibleRecipientsForRemainingGiver, ...
```